scripts/build.py

```diff
@@ -96,30 +96,54 @@ def copy_files_source_to_target(source, target):
 
         for filename in files:
             source_path = os.path.join(root, filename)
-            if source_path.find(target) > 0:
+            if target in source_path:
                 continue
-            if source_path.find(source) == -1:
+            if source not in source_path:
                 continue
             target_path = source_path.replace(
                 source, target)
             ensure_directory_exists(target_path)
             shutil.copyfile(source_path, target_path)
 
 
-def _build_files():
-    ensure_directory_exists(OUT_DIR)
-    shutil.rmtree(OUT_DIR)
+def build_files(source, target, ignore=None):
+    """Minifies all css and js files, and removes whitespace from html in source
+    directory and copies it to target, ignoring paths/files mentioned in ignore.
+    Copies files in ignore to target without any changes.
+    Arguments:
+        source, target: strings
+        ignore: list of files/paths to ignore
+    """
+    print 'Processing %s' % os.path.join(os.getcwd(), source)
+    print 'Generating into %s' % os.path.join(os.getcwd(), target)
+    ensure_directory_exists(target)
+    shutil.rmtree(target)
 
-    for root, dirs, files in os.walk(os.path.join(os.getcwd(), HEAD_DIR)):
+    for root, dirs, files in os.walk(os.path.join(os.getcwd(), source)):
         for directory in dirs:
             print 'Processing %s' % os.path.join(root, directory)
         for filename in files:
             source_path = os.path.join(root, filename)
-            if source_path.find(OUT_DIR) > 0:
+            if target in source_path:
+                continue
+            if source not in source_path:
                 continue
-            if source_path.find(HEAD_DIR) == -1:
+            target_path = source_path.replace(source, target)
+
+            only_copy_file = False
+            # Only copy files in ignore or with extensions mentioned in
+            # FILE_EXTENSIONS_TO_COPY_WITHOUT_MINIFICATION variable.
+            # We do not copy any py scripts.
```

Comment:
Is this comment necessary? The code doesn't seem to be special casing this, so I'm a bit confused why it's here.